src/main/kotlin/com/vk/tlschema/highlighting/TLSchemaBreadcrumbsProvider.kt

```diff
@@ -0,0 +1,39 @@
+package com.vk.tlschema.highlighting
+
+import com.intellij.lang.Language
+import com.intellij.openapi.util.NlsSafe
+import com.intellij.psi.PsiElement
+import com.intellij.ui.breadcrumbs.BreadcrumbsProvider
+import com.vk.tlschema.TLSchemaLanguage
+import com.vk.tlschema.psi.TLSchemaConstructorDeclarations
+import com.vk.tlschema.psi.TLSchemaDeclaration
+import com.vk.tlschema.psi.TLSchemaFunDeclarations
+import java.util.*
+
+class TLSchemaBreadcrumbsProvider : BreadcrumbsProvider {
+    override fun getLanguages() = arrayOf(TLSchemaLanguage.INSTANCE)
+
+    override fun acceptElement(psiElement: PsiElement): Boolean {
```

Comment:
change `PsiElement` to `element`, as in the [BreadcrumbsProvider](https://github.com/JetBrains/intellij-community/blob/99699ea301d9986681e6420197c9318008112c79/platform/editor-ui-api/src/com/intellij/ui/breadcrumbs/BreadcrumbsProvider.java#L32) interface

Reply:
renamed